communication.rst

```diff
@@ -99,21 +99,89 @@ We have our own `Discourse`_ forum for both developers and users. This forum
 complements the `python-dev`_, `python-ideas`_, `python-help`_, and
 `python-list`_ mailing lists.
 
-Most core development discussions take place in the open forum categories for
-`PEPs`_ and `Core Development`_. (These are the Discourse equivalents to the
-python-dev mailing list)
-
-The `Committers`_ category restricts posting to core developers only, and is
-used more for announcements and notifications, rather than for discussions. It
-is also the venue for core developer promotion votes. (This category is the
-equivalent of the python-committers mailing list)
-
-There is a final, rarely used, core development category on Discourse that is
-only visible to core developers. This can be used to share administrative
-information with all core developers in a non-public forum (e.g. logistics for
-in person core development sprints), as well as for individual core developers
-to share info that they'd like other core devs to be aware of, but would prefer
-not to have permanently archived on the internet.
+This forum has different categories and most core development discussions
+take place in the open forum categories for `PEPs`_ and `Core Development`_
+(these are the Discourse equivalents to the python-dev mailing list).
+All categories are open for users to read and post with the exception of
+the Committers category, where posting is restricted to the `CPython
+<https://github.com/python/cpython>`_ core developers.
+
+The Committers category is often used for announcements and notifications.
+It is also a common venue for the core developer promotion votes (this
+category is equivalent to the python-committers mailing list).
+
+Tutorials for new users
+'''''''''''''''''''''''
+To start a topic or participate in any discussions in the forum, sign up and
+create an account using an email address or GitHub account. You can do so by
+clicking the "Sign Up" button on the top right hand corner of the `Discourse`_
+main page.
+
+The Python Discourse `Quick Start <https://discuss.python.org/t/python-discourse-quick-start/116>`_
+compiled by `Carol Willing <https://discuss.python.org/u/willingc/>`_ gives you
+a quick overview on how to kick off Python Discourse.
+
+We recommend new users getting familiarised with the forum by going through Discobot tutorials.
+These tutorials can be activated by replying to a welcome message from "discourse
+Greetings!" received under Notifications and Messages in your user account.
+
+* Click on your personal account found on the top right hand corner of the page.
+* The dropdown menu will show four different icons: 🔔 (Notifications),
+  🔖 (Bookmarks), ✉️ (Messages), and 👤 (Preferences).
+* Select either Notifications or Messages.
+* Open the "Greetings!" message sent by Discobot to start the tutorial.
+* Ensure that you read through the `Python Code of Conduct <https://discuss.python.org/faq>`_.
+  We are to be open, considerate and respectful to all users in the community.
```

Comment:
I think this would be better as a standalone sentence separated from the list that explains how to launch the discobot tutorial.
```suggestion

Ensure that you read through the `Python Code of Conduct <https://discuss.python.org/faq>`_.
We are to be open, considerate and respectful to all users in the community.
```